sdk/core/azure-core/inc/http/policy.hpp

```diff
@@ -7,6 +7,9 @@
 #include "context.hpp"
 #include "http.hpp"
 #include "transport.hpp"
+#include "uuid.hpp"
+
+#include <random>
```

Comment:
We should move `#include <random>` into `uuid.hpp`. Because this file doesn't depend on `<random>`, `uuid.hpp` does.

Reply:
Good catch, I'll make that update